src/ui/public/vislib/lib/axis/axis_config.js

```diff
@@ -132,6 +132,10 @@ export default function AxisConfigFactory() {
         this._values.labels.truncate = _.get(axisConfigArgs, 'labels.truncate', 100);
       }
 
+      if (this.get('type') === 'category' && !this.isHorizontal()) {
+        this._values.scale.inverted = _.get(axisConfigArgs, 'scale.inverted', true);
```

Comment:
Out of curiosity: Is there a way to create a situation in which `axisConfigArgs.scale.inverted === true` via the UI?